common/static/css/edx-cc.css

```diff
@@ -0,0 +1,63 @@
+@font-face {
+  font-family: 'edx-cc';
+  src: url('../fonts/edx-cc/edx-cc.eot?52318265');
+  src: url('../fonts/edx-cc/edx-cc.eot?52318265#iefix') format('embedded-opentype'),
+       url('../fonts/edx-cc/edx-cc.woff?52318265') format('woff'),
+       url('../fonts/edx-cc/edx-cc.ttf?52318265') format('truetype'),
+       url('../fonts/edx-cc/edx-cc.svg?52318265#edx-cc') format('svg');
+  font-weight: normal;
+  font-style: normal;
+}
+/* Chrome hack: SVG is rendered more smooth in Windozze. 100% magic, uncomment if you need it. */
+/* Note, that will break hinting! In other OS-es font will be not as sharp as it could be */
```

Comment:
The sandbox keeps failing to build with the following 2 errors - I don't know how to resolve it. I've reached out to a few people to help but putting it here too. https://gist.githubusercontent.com/sarina/9222b41ce12663723066/raw/5353cf007073a86a3540aaf355b22b1af35eac3c/gistfile1.txt